src/main/java/Lovie.java

```diff
@@ -0,0 +1,103 @@
+        System.out.println("        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~" + 
+                "\n\n");
+
+        while (true) {
+            input = inputGetter.nextLine(); 
+            if (input.equals("bye")) {
+                break; 
+            } else if (input.equals("list")) {
+                System.out.println("        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~" + 
+                        "~~~~~~~~~~~~~~\n\n");
+                int counter = 0; 
+                while (counter < tasksList.size()) {
+                    String listNumber = Integer.toString(counter + 1); 
+                    System.out.println("        " + listNumber + ". " + "[" + 
+                            tasksList.get(counter).getStatusIcon() + "] " + 
+                            tasksList.get(counter).description + "\n\n");
+                    counter += 1; 
+                }
+                System.out.println("        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~" + 
+                        "~~~~~~~~~~~~~~~\n\n");
+            } else if (input.contains("unmark")) {
+                int taskNumber = Integer.parseInt(input.split(" ")[1].toString()) - 1; 
+                if (taskNumber >= tasksList.size() || taskNumber <= 1) {
+                    System.out.println("\n\n");
+                    System.out.println("        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~" + 
+                            "~~~~~~~~~~~~~~~~~~~~~~~\n\n");
+                    System.out.println("        Sorry, there is no record of a task number " + 
+                            input.split(" ")[1].toString() + "\n");
+                    System.out.println("        Can I help you with anything else?\n\n"); 
+                    System.out.println("        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~" + 
+                            "~~~~~~~~~~~~~~~~~~~~~~~~\n\n"); 
+                } else {
+                    Task selectedTask = tasksList.get(taskNumber); 
+                    selectedTask.markAsUndone();
+                    System.out.println("\n\n");
+                    System.out.println("        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~" + 
+                            "~~~~~~~~~~~~~~~~~~~~~~\n\n");
+                    System.out.println("        Okay, no worries. I've unmarked this task for you: \n"); 
+                    System.out.println("            [" + selectedTask.getStatusIcon() + "] " + 
+                            selectedTask.description + "\n"); 
+                    System.out.println("        What else can I do for you today?\n\n"); 
+                    System.out.println("        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~" + 
+                            "~~~~~~~~~~~~~~~~~~~~~~\n\n"); 
+                }
+            } else if (input.contains("mark")) {
+                int taskNumber = Integer.parseInt(input.split(" ")[1].toString()) - 1; 
```

Comment:
Perhaps better to have  a white space after naming the variable (taskNumber) and the if/else statement.